src/commands/test/lib/included-files-parser.ts

```diff
@@ -1,8 +1,32 @@
-import { IFileDescriptionJson } from "./test-manifest-reader";
+import { IFileDescriptionJson, ITestCloudManifestJson } from "./test-manifest-reader";
 import * as path from "path";
+import * as pfs from "../../../util/misc/promisfied-fs";
+import { validateXmlFile } from "./xml-util";
+import { out } from "../../../util/interaction";
 
 const invalidCharactersRegexp = /['"!#$%&+^<=>`|]/;
 
+export async function copyIncludedFiles(manifest: ITestCloudManifestJson, include: string[], rootDir: string) {
```

Comment:
Are there any tests for this stuff?

Reply:
They were sadly lacking on the original logic - I can add some if you need me to?